bentoml/_internal/frameworks/transformers.py

```diff
@@ -160,7 +190,35 @@ def save_model(
         name (:code:`str`):
             Name for given model instance. This should pass Python identifier check.
         pipeline (:code:`Pipeline`):
-            Instance of the Transformers pipeline to be saved.
+            Instance of the Transformers pipeline to be saved. See Transformers
+            ``module src/transformers/pipelines/__init__.py`` for more details.
+        task_name (:code:`str`):
+            Name of pipeline task. If not provided, the task name will be derived from ``pipeline.task``.
+            ``task_name`` and ``task_definition`` must be both provided or both not provided.
```

Comment:
maybe another way to say this is

if task_name is provided, then task_definition is also required.